showkase/build.gradle

```diff
@@ -14,13 +14,6 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
         consumerProguardFiles "consumer-rules.pro"
     }
-
-    buildTypes {
```

Comment:
I'm not 100% sure that this is what was causing the issue but think this might have been the cause...